src/handlers/serenity.rs

```diff
@@ -17,45 +22,102 @@ use crate::commands::{
     remove::*, repeat::*, resume::*, seek::*, shuffle::*, skip::*, stop::*, summon::*, version::*,
 };
 
+const ROLE: &str = "Parrot DJ";
```

Comment:
nitpick: Is it possible to get the bot's name here instead of simply "Parrot DJ"?

Reply:
The problem is that the bot's name is already its own role and can't be assigned to users. I also don't like this default but have no better idea. Thankfully it is also changeable by the Server Admin.

![image](https://user-images.githubusercontent.com/16060539/151027489-df3ce0a9-43b0-492f-8537-080987de97a8.png)

Reply:
I meant something like "${botName}'s DJ".

Reply:
Seems like a good idea I will look into it